pimd/pim_msdp.c

```diff
@@ -1432,34 +1335,10 @@ static void pim_msdp_mg_mbr_do_del(struct pim_msdp_mg *mg,
 	}
 }
 
-enum pim_msdp_err pim_msdp_mg_mbr_del(struct pim_instance *pim,
-				      const char *mesh_group_name,
-				      struct in_addr mbr_ip)
-{
-	struct pim_msdp_mg_mbr *mbr;
-	struct pim_msdp_mg *mg = pim->msdp.mg;
-
-	if (!mg || strcmp(mg->mesh_group_name, mesh_group_name)) {
-		return PIM_MSDP_ERR_NO_MG;
-	}
-
-	mbr = pim_msdp_mg_mbr_find(pim, mbr_ip);
-	if (!mbr) {
-		return PIM_MSDP_ERR_NO_MG_MBR;
-	}
-
-	pim_msdp_mg_mbr_do_del(mg, mbr);
-	/* if there are no references to the mg free it */
-	pim_msdp_mg_free(pim);
-
-	return PIM_MSDP_ERR_NONE;
-}
-
-static void pim_msdp_mg_src_do_del(struct pim_instance *pim)
```

Comment:
Same suggestion here -
1. rename pim_msdp_change_source to pim_msdp_mg_src_add
2. rename pim_msdp_mg_src_do_del to pim_msdp_src_del
to create fn_name symmetry